workers/data_refinery_workers/processors/test_qn_reference.py

```diff
@@ -251,3 +252,32 @@ def test_qn_management_command(self):
         # There's not enough samples available in this scenario so we
         # shouldn't have even made a processor job.
         self.assertEqual(ProcessorJob.objects.count(), 0)
+
+    @tag("qn")
+    def test_qn_target_without_updating_organism(self):
+        """In the cron job tests, we want to create a new QN target without
+        actually setting it as the new default for the experiment."""
```

Comment:
```suggestion
        """Test that create_results=False will not update the experiment.
        
        This is because in the cron job tests, we want to create a new QN target without
        actually setting it as the new default for the experiment."""
```